std/machines/hash/keccakf.asm

```diff
@@ -0,0 +1,8 @@
+// TODO: placeholder for Rust wrapper
+machine KeccakF with
+    latch: LATCH,
+{ 
+    let x;
+    let y;
+    operation keccakf x -> y;
```

Comment:
i think this will be `keccakf x, y ->`, that is, it takes two memory addresses as input (the second one will be written to by the machine)

Reply:
It also needs the current time step! The main machine already has a `STEP` fixed column that can be passed as an argument.

Reply:
> i think this will be `keccakf x, y ->`, that is, it takes two memory addresses as input (the second one will be written to by the machine)

Agreed and done.

Reply:
`instr keccakf X, Y -> ~ keccakf X, Y, STEP;`

Reply:
Main machine -> keccak .asm